src/uct/sm/mm/base/mm_ep.c

```diff
@@ -302,7 +303,9 @@ static UCS_F_ALWAYS_INLINE ssize_t uct_mm_ep_am_common_send(
         break;
     case UCT_MM_SEND_AM_SHORT_IOV:
         elem_flags   = UCT_MM_FIFO_ELEM_FLAG_INLINE;
-        elem->length = uct_iov_to_buffer(iov, iovcnt, 0, elem + 1, SIZE_MAX);
+        ucs_iov_iter_init(&iov_iter);
+        elem->length = uct_iov_to_buffer(iov, iovcnt, &iov_iter, elem + 1,
+                                         SIZE_MAX);
```

Comment:
can you pls check if this change affects am_short_iov performance?

Reply:
I've checked latency of sysv transport using 16- and 24- byte messages before and after changing uct_iov_to_buffer.
I haven't found any notable difference between the versions.